appsdk/sdk.go

```diff
@@ -62,6 +64,10 @@ const (
 	ProfileSuffixPlaceholder = "<profile>"
 	envV1Profile             = "edgex_profile" // TODO: Remove for release v2.0.0
 	envProfile               = "EDGEX_PROFILE"
+	envV1Service             = "edgex_service"    // deprecated TODO: Remove for release v2.0.0
+	envServiceProtocol       = "Service_Protocol" // Used for envV1Service processing TODO: Remove for release v2.0.0
+	envServiceHost           = "Service_Host"     // Used for envV1Service processing TODO: Remove for release v2.0.0
+	envServicePort           = "Service_Port"     // Used for envV1Service processing TODO: Remove for release v2.0.0
```

Comment:
do you wanna make these issues so we dont forget about them?

Reply:
Sure.